README.md

```diff
@@ -1,51 +1,87 @@
+---
 
-The aim of this project is to create a module-safe, lightweight translation library for a node environment, based on similar concepts to i18next.
+## Features 
 
-## Install
+### Translation Management
+- **JSON-based storage**: Simple key/value pairs in structured JSON files
+- **Nested keys**: Organize translations hierarchically (e.g., `"buttons.submit": "Submit"`)
+
+### Locale Handling
+- **Automatic detection**: Supports multiple lookup sources:
+    - 🍪 Cookies
+    - 🔗 Query parameters
+    - 🌐 HTTP headers (`Accept-Language`)
+- **Express middleware**: Zero-config integration for route handling
+- **Multi-source loading**: Load translations from project directories and `node_modules`
+
+## Installation 
 
 ```
 npm install hmpo-i18n
 ```
 
-## Usage
+## Basic Setup
 
-First create some resource files. These should be json or yaml files and the location of the file within your project will define the language it corresponds to.
+#### 📁 Folder Structure
+```json
+your-project/
+├── app.js
+├── views/
+│   └── index.html
+└── locales/
+    ├── en/
+    │   └── default.json
+    └── cy/
+        └── default.json
+```
 
-An example file structure would look like:
+### Create translation files
 
+```json
+// locales/en/default.json
+{
+  "welcome": {
+    "title": "Welcome to Our Service",
+    "message": "This is a random message"
+  },
+  "errors": {
+    "notFound": "Page not found",
+    "serverError": "Something went wrong"
+  }
+}
 ```
-index.js
-/locales
-  /en
-    /default.json
-    /validation.yaml
-    /pages.errors.yaml
-  /fr
-    /default.json
-  /de
-    /default.json
+
+### Pre-defined resources
+
+You can manually define resource sets at initialisation by passing in an object with the `resources` parameter of the options. This is useful when you have shared translations that are used across a number of modules, so you may wish to load from npm modules or similar - Default: `{}`
+
+```javascript
+i18n({
+    resources: {
+      en: require('shared-translations').en,
+      fr: require('shared-translations').fr
+    }
+});
 ```
 
-If you wish to create additional namespaces within your project, then create additional files within a language directory with names corresponding to the namespace. For details on how to configure resource paths see ["Resource path" documentation below](#resource-path)
-These namespaces are superimposed onto the default layout.
-### Standalone:
+## Usage 
+
+### 1. As a standalone:
 
 ```javascript
 var i18n = require('hmpo-i18n');
 
 // i18n fires a "ready" event when it is done loading resources
 i18n.on('ready', function () {
-    i18n.translate('name.first', 'fr');
+    i18n.translate('welcome.title', 'en');
 });
 ```
 
-### As express middleware:
+### 2. As an express middleware:
```

Comment:
Might be worth splitting example as it was before to show it can be used as express middleware using translate and then passing custom language detection